src/MimePart.php

```diff
@@ -464,15 +464,15 @@ private function copyContentStream($fromHandle, $toHandle, $isUUEncoded)
         rewind($fromHandle);
         // changed from stream_copy_to_stream because hhvm seems to stop before
         // end of file for some reason
-        while (($read = fread($fromHandle, 1024)) != false) {
+        while (($read = fread($fromHandle, 1024)) !== false) {
```

Comment:
@MatthiasKuehneEllerhold 

Travis is stalling.  Probably from this change :smile: